### PR TITLE
feat: add dde-launchpad item-arrangement config

### PR DESCRIPTION
**Pull request by yixinshark**
app order and default folder config

Log: add dde-launchpad item-arrangement
Influence: launchpad app order and default folder

### CODE REVIEW

File: skel/.config/deepin/dde-launchpad/item-arrangement.ini [hunk 1]
```diff
@@ -0,0 +1,10 @@
+[fullscreen]
+1\name=Libreoffice
```
Comment (BLumia) [inline skel/.config/deepin/dde-launchpad/item-arrangement.ini:2]:
```suggestion
1\name=LibreOffice
```

File: skel/.config/deepin/dde-launchpad/item-arrangement.ini [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+1\pageCount=1
+1\pageItems\0=libreoffice-writer.desktop, libreoffice-base.desktop, libreoffice-calc.desktop, libreoffice-draw.desktop, libreoffice-impress.desktop, libreoffice-math.desktop, libreoffice-startcenter.desktop
+2\name=Fcitx5
```
Comment (BLumia) [inline skel/.config/deepin/dde-launchpad/item-arrangement.ini:5]:
```suggestion
2\name=Fcitx 5
```

File: skel/.config/deepin/dde-launchpad/item-arrangement.ini (hunk 1 above)
Comment (yixinshark) [reply skel/.config/deepin/dde-launchpad/item-arrangement.ini]: done

File: skel/.config/deepin/dde-launchpad/item-arrangement.ini (hunk 2 above)
Comment (yixinshark) [reply skel/.config/deepin/dde-launchpad/item-arrangement.ini]: done

**Review by BLumia — APPROVED** [review]

**Pull request merged**